### PR TITLE
[rt-smart][Kconfig] Fix Kconfig issue for rt-smart

### PR DESCRIPTION
**Pull request by BernardXiong**
## 拉取/合并请求描述：(PR description)

fix Kconfig issue for AArch64 in rt-smart.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [ ] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [ ] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [ ] 代码是高质量的 Code in this PR is of high quality
- [ ] 已经使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

File: bsp/qemu-virt64-aarch64/Kconfig [hunk 1]
```diff
@@ -28,3 +28,17 @@ config SOC_VIRT64_AARCH64
...
+
+config BOARD_USING_SMART:
+    bool "Enable rt-smart features"
+    select RT_USING_SMART
+    select RT_USING_SIGNALS
+    default n
+
+if RT_USING_SMART
+    config KERNEL_VADDR_START
+        default 0xffff000000000000
```
Comment (Guozhanxin) [inline bsp/qemu-virt64-aarch64/Kconfig:40]: 我之前是这么改了一下，你这种再定义一份的话，会冲突
image

Comment (BernardXiong) [reply bsp/qemu-virt64-aarch64/Kconfig:40]: 这里是bsp里直接定义，是否可以把其它的定义给覆盖掉?

Comment (Guozhanxin) [reply bsp/qemu-virt64-aarch64/Kconfig:40]: kconfig 不允许一个选项在不同地方重复定义。

**Comment by Guozhanxin** [comment]
已在另外pr解决此问题，关闭此pr

**Pull request closed without merging**